porch/pkg/registry/porch/packagecommon.go

```diff
@@ -72,7 +72,8 @@ func (r *packageCommon) listPackageRevisions(ctx context.Context, filter package
 
 		revisions, err := r.cad.ListPackageRevisions(ctx, repositoryObj, filter.ListPackageRevisionFilter)
 		if err != nil {
-			return err
+			klog.Warningf("error listing package revisions from repository %s/%s: %s", repositoryObj.GetNamespace(), repositoryObj.GetName(), err)
```

Comment:
Should this be `klog.Errorf`?

Reply:
Not sure...what's our criteria there? We are not aborting.

Reply:
I think this will typically happen if we register a repo before it exists, or there is an auth error, networking issues, etc.

Reply:
Thanks for fixing this! I think a warning seems reasonable here. Might be easier to assess if we had an example output as part of the PR.

Reply:
Some examples:

```
[hi on] jbelamaric@jbelamaric:~/nephio-install/config-control$ grep "error listing" /tmp/j | head
W0912 23:05:25.651537       1 packagecommon.go:75] error listing package revisions from repository default/edge01: cannot fetch repository default/edge01: repository not found
W0912 23:05:26.221433       1 packagecommon.go:75] error listing package revisions from repository default/edge02: cannot fetch repository default/edge02: repository not found
W0912 23:05:28.414151       1 packagecommon.go:75] error listing package revisions from repository default/edge01: cannot fetch repository default/edge01: repository not found
W0912 23:05:28.922793       1 packagecommon.go:75] error listing package revisions from repository default/edge02: cannot fetch repository default/edge02: repository not found
W0912 23:05:35.794923       1 packagecommon.go:75] error listing package revisions from repository default/edge01: cannot fetch repository default/edge01: repository not found
W0912 23:05:36.380900       1 packagecommon.go:75] error listing package revisions from repository default/edge02: cannot fetch repository default/edge02: repository not found
W0912 23:05:38.597123       1 packagecommon.go:75] error listing package revisions from repository default/edge01: cannot fetch repository default/edge01: repository not found
W0912 23:05:39.151027       1 packagecommon.go:75] error listing package revisions from repository default/edge02: cannot fetch repository default/edge02: repository not found
W0912 23:07:21.692834       1 packagecommon.go:75] error listing package revisions from repository default/edge01: cannot fetch repository default/edge01: repository not found
W0912 23:07:22.132945       1 packagecommon.go:75] error listing package revisions from repository default/edge02: cannot fetch repository default/edge02: repository not found
[hi on] jbelamaric@jbelamaric:~/nephio-install/config-control$ grep "error listing" /tmp/jj | head
W0913 11:54:57.002933       1 packagecommon.go:75] error listing package revisions from repository default/nephio-example-packages: client rate limiter Wait returned an error: context canceled
W0913 11:54:57.021899       1 packagecommon.go:75] error listing package revisions from repository default/blueprints-nephio-gcp: client rate limiter Wait returned an error: context canceled
W0913 11:54:57.022383       1 packagecommon.go:75] error listing package revisions from repository default/config-control: client rate limiter Wait returned an error: context canceled
W0913 11:55:12.708364       1 packagecommon.go:75] error listing package revisions from repository default/free5gc-packages: client rate limiter Wait returned an error: context canceled
W0913 11:55:12.708829       1 packagecommon.go:75] error listing package revisions from repository default/nephio: client rate limiter Wait returned an error: context canceled
W0913 11:58:28.197907       1 packagecommon.go:75] error listing package revisions from repository default/nephio-example-packages: client rate limiter Wait returned an error: context canceled
W0913 11:58:33.206931       1 packagecommon.go:75] error listing package revisions from repository default/config-control: client rate limiter Wait returned an error: context canceled
W0913 11:58:48.987658       1 packagecommon.go:75] error listing package revisions from repository default/blueprints-nephio-gcp: client rate limiter Wait returned an error: context canceled
W0913 11:58:48.988718       1 packagecommon.go:75] error listing package revisions from repository default/config-control: client rate limiter Wait returned an error: context canceled
W0913 11:59:08.546775       1 packagecommon.go:75] error listing package revisions from repository default/free5gc-packages: client rate limiter Wait returned an error: context canceled
[hi on] jbelamaric@jbelamaric:~/nephio-install/config-control$ 
```

Does that help? I don't think in general we would want to, for example, page someone due to these sort of things. The error with the missing repo is simply a user error - or maybe not even an error, but just registering the repo in advance.